cmd/common_test.go

```diff
@@ -36,6 +38,11 @@ func executeCli(cli string) []string {
 	return strings.Split(so.String(), "\n")
 }
 
+func getRandom() uint64 {
+	rand.Seed(time.Now().UTC().Unix())
```

Comment:
Just a comment. Instead of returning a uint64, you can just return `time.Now().Unix()` as a string that can be added to the files. That is what is done in https://github.com/libopenstorage/sdk-test/blob/master/pkg/sanity/volume.go#L78